yarn.lock

```diff
@@ -24605,10 +24605,10 @@ table@^6.0.4:
     string-width "^4.2.3"
     strip-ansi "^6.0.1"
 
-tabster@^1.4.0:
-  version "1.4.0"
-  resolved "https://registry.yarnpkg.com/tabster/-/tabster-1.4.0.tgz#d3be9b34348433970a0b4a593c5d6edc00a6df4f"
-  integrity sha512-Al7lJSJ4xT3hw4NfYE5/jhCvNeN2wonRyc87Y+J22XMVDvlS4Rc6ivsPxK0dYYGWmpucAg8tNAgctaAochAuWA==
+tabster@^1.4.2:
```

Comment:
curious if this bump was needed for this PR ? asking so we avoid issues as tabster introduced BC in the past on minor bumps (in dom structure injections)

Reply:
the bump is necessary in order to pick up the changes that enable the older focus trap behaviour